kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationPerConnectionSuite.scala

```diff
@@ -250,7 +250,7 @@ class KyuubiOperationPerConnectionSuite extends WithKyuubiServer with HiveJDBCTe
         val executeStmtResp = client.ExecuteStatement(executeStmtReq)
         assert(executeStmtResp.getStatus.getStatusCode === TStatusCode.ERROR_STATUS)
         val elapsedTime = System.currentTimeMillis() - startTime
-        assert(elapsedTime > 3 * 1000 && elapsedTime < 20 * 1000)
+        assert(elapsedTime > 10 * 1000 && elapsedTime < 20 * 1000)
```

Comment:
keep it the same with before?

Reply:
If we modify `ENGINE_ALIVE_TIMEOUT`, it should also be modified here.

I now change it to 3000.